sdk/synapse/Azure.Analytics.Synapse.Artifacts/src/Azure.Analytics.Synapse.Artifacts.csproj

```diff
@@ -38,5 +37,8 @@
     <Compile Include="$(AzureCoreSharedSources)TaskExtensions.cs" LinkBase="Shared" />
     <Compile Include="$(AzureCoreSharedSources)AzureResourceProviderNamespaceAttribute.cs" LinkBase="Shared" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
Let's keep the package reference in this one, as it's not yet an LLC library.

Reply:
I removed the PackageReference and added ProjectRerence because in Generator.Shared code, it was giving me this error:
![image](https://user-images.githubusercontent.com/45216704/139957694-7b9f1c02-dbd3-4f55-8625-58dbef66a740.png)

Reply:
Cool, ship it!

Reply:
Now that we have added the Experimental condition in Generator.Shared code, I've added the package reference back and removed the project reference.